src/app/parts-editor/components/dialog-database/dialog-database.component.html

```diff
@@ -0,0 +1,20 @@
+<h1 i18n>Parts missing from database</h1>
+
+<form [formGroup]="missingPartsForm" (ngSubmit)="onSubmit()">
```

Comment:
NgSubmit is a part of FormsModule, not ReactivePartsModule. Also semantically submitting is not correct here. submitting a form means sending POST request to server. 

Do not use type submit on a button without clear reason. Use standard click handler on a button, and only close dialog when the underlying form control is valid